requirements.txt

```diff
@@ -1,4 +1,5 @@
 PyYAML>=5.3
+dm-tree
```

Comment:
As noted above, wondering if we can live without introducing a dependency into the core classes of this library. It looks really handy, but that couples our core abstractions to external code.